.github/workflows/ci.yml

```
@@ -17,6 +17,19 @@ on:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+
+    - name: Get dependencies
+      run: make deps
+
+    - name: Run jsonnetfmt
+      run: for t in templates/**/*.jsonnet; do echo "Testing template ${t}"; ./bin/jsonnetfmt --test $t; echo 'Results:' ${?}; done
```

We can update the `format` target from the makefile to include some comments so we can reduce this expression to:

```suggestion
    - name: Get dependencies
      run: make format
```